docs/getting-started/cli.md

```diff
@@ -0,0 +1,198 @@
+---
+slug: cli
+title: CLI-only workflow (Pro)
+hide_title: true
+description: Learn how to set up Semgrep, scan your first project for security issues, and view your findings in the CLI.
+tags:
+  - quickstart
+  - cli
+---
+
+import MoreHelp from "/src/components/MoreHelp";
+import Install from "/src/components/procedure/_install-cli.mdx";
+import Login from "/src/components/procedure/_login-activate.mdx";
+
+# CLI-only workflow with Semgrep's Pro Engine
+
+Learn how to set up Semgrep, scan your project for security issues using Semgrep's Pro Engine, and view your findings in the CLI.
+
+:::info
+For scans in your local development environment, `semgrep scan` is the preferred method to scan with Semgrep, especially for writing and testing custom rules.
+:::
+
+## Prerequisites
+
+Before proceeding:
+
+* See [Prerequisites](/prerequisites/) to ensure that your machine meets Semgrep's requirements.
+* Ensure that you have and are logged into your [Semgrep Account](https://semgrep.dev/login).
+* Ensure that you've enabled the **Pro Engine** <i class="fa-solid fa-toggle-large-on"></i> toggle on Semgrep Cloud Platform's [Settings](https://semgrep.dev/orgs/-/settings) page.
+
+## Set up Semgrep
+
+<Install />
+
+## Log into your Semgrep account
+
+<Login />
+
+You must log in to your Semgrep account before proceeding. Otherwise, you can't use `semgrep ci`.
```

Comment:
I thought we weren't going to say this just yet? @s-santillan

Reply:
I don't mind removing, but it already warns users of this in the CLI; I thought it might be helpful to get ahead of the not-logged-in error?

Reply:
Yeah the product already enforces it. We'd just be matching the current product behavior. Katie noticed it first; me not noticing this ASAP and realizing how much it impacts the docs is something I have to work on.

Reply:
@s-santillan Based on the latest thread, my understanding is that it doesn't enforce it yet (although it is something that may happen in the future). What it enforces now is that you can't both be logged in/provide a token, and use --config. If you're not logged in and use --config, it still works.

Reply:
Got it. This isn't in live yet, so I'll make the correction in my follow up PR (https://github.com/semgrep/semgrep-docs/pull/1350). cc @khorne3 (And yes I will fix the sidebars.)